guides/source/routing.md

```diff
@@ -18,51 +18,66 @@ After reading this guide, you will know:
 The Purpose of the Rails Router
 -------------------------------
 
-The Rails router recognizes URLs and dispatches them to a controller's action, or to a Rack application. It can also generate paths and URLs, avoiding the need to hardcode strings in your views.
+The Rails router matches incoming HTTP requests to specific controller actions
+in your Rails application based on the URL path. (It can also forward to a
+[Rack](rails_on_rack.html) application.) The router also generates path and URL
+helpers based on the resources configured in the router.
 
-### Connecting URLs to Code
+### Routing Incoming URLs to Code
 
-When your Rails application receives an incoming request for:
+When your Rails application receives an incoming request, it asks the router to match it to a controller action. For example, take the following incoming request:
 
 ```
-GET /patients/17
+GET /users/17
 ```
 
-it asks the router to match it to a controller action. If the first matching route is:
+The Router's job is to match it to a controller action. If the first matching route is:
 
 ```ruby
-get '/patients/:id', to: 'patients#show'
+get '/users/:id', to: 'user#show'
 ```
 
-the request is dispatched to the `patients` controller's `show` action with `{ id: '17' }` in `params`.
+The request is matched to the `UsersController` class's `show` action with `{ id: '17' }` in the `params` hash.
```

Comment:
Since we change the naming of `controller` to `class` I wonder if we shouldn't change also `action` to `method` since that's in same paradigm.

Reply:
Suggestion to reword: "to the `UsersController` class's `show` action" -> "to the `show` action in `users` controller with...". This keeps it in the Rails paradigm.